securesystemslib/signer/_sslib_signer.py

```diff
@@ -99,6 +100,10 @@ def from_priv_key_uri(
 
         return cls(keydict)
 
+    @property
+    def public_key(self) -> Key:
+        return self._public_key
```

Comment:
Since sslibsigner is just a wrapper it could just return the underlying `self._crypto_signer.public_key` without storing a copy itself... but I think this is fine too: the whole class is deprecated anyway, we'll probably remove it very soon.